html/index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<!-- Meta -->
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="description" content="Bugmark is the fastest way to build issue futures into your App.">
+
+	<!-- Twitter Cards -->
+	<meta name="twitter:card" content="summary">
+	<meta name="twitter:site" content="https://bugmark.net">
+	<meta name="twitter:title" content="Bugmark">
+	<meta name="twitter:description" content="Bugmark is the fastest way to build issue futures into your App.">
+
+	<!-- Facebook Open Graph -->
+	<meta property="og:locale" content="en_US">
+	<meta property="og:url" content="https://bugmark.net">
+	<meta property="og:site_name" content="Bugmark">
+	<meta property="og:title" content="Bugmark">
+	<meta property="og:description" content="Bugmark is the fastest way to build issue futures into your App.">
+	<meta property="og:type" content="website">
+
+	<!-- Title -->
+	<title>Bugmark</title>
+
+	<!-- Fonts -->
+	<link href="https://use.fontawesome.com/releases/v5.0.8/css/all.css" rel="stylesheet">
+
+	<!-- CSS -->
+	<link rel="stylesheet" href="assets/vendor/bootstrap/dist/css/bootstrap.min.css">
+</head>
+
+<body>
+	<!-- navigation -->
+	<nav class="navbar navbar-light">
+		<div class="container">
+			<a class="navbar-brand h1" href="#">Bugmark</a>
+			<button class="navbar-toggler collapsed" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+				<span class="navbar-toggler-icon"></span>
+			</button>
+			<div class="collapse navbar-collapse show" id="navbarNav">
+				<ul class="nav">
+					<li class="nav-item"><a class="nav-link" href="#">App</a></li>
+					<li class="nav-item"><a class="nav-link" href="#">Blog</a></li>
+					<li class="nav-item"><a class="nav-link" href="#">Docs</a></li>
+					<li class="nav-item"><a class="nav-link" href="#">Development</a></li>
+				</ul>
+			</div>
+		</div>
+	</nav>
+	<!-- //navigation -->
+	<!-- main -->
+	<main role="main">
+		<!-- hero -->
+		<section id="hero" class="jumbotron jumbotron-fluid mb-0 text-center">
+			<div class="container">
+				<h1 class="display-4 pt-5 pb-5">The world's fastest way to integrate issue futures into your app</h1>
+				<a class="btn btn-lg btn-outline-primary" href="">Bugmark.net</a>
```

Comment:
We could add a scroll link to the section "What is Bugmark?" in this button. In the mockup, the link point to bugmark.net, but I believe when we finished here this will be the new site, am I wrong?